src/main/scala/org/globalforestwatch/features/FireAlertFeature.scala

```diff
@@ -0,0 +1,62 @@
+package org.globalforestwatch.features
+
+import geotrellis.vector
+import org.globalforestwatch.util.GeometryReducer
+import geotrellis.vector.Geometry
+import org.apache.spark.sql.Row
+
+object FireAlertFeature extends Feature {
+  def getFireAlertFeature(fireAlertType: String, x: Double, y: Double, i: Array[String], featureId: FeatureId): vector.Feature[Geometry, FeatureId] = {
+    val adjustedX =
+      if (x.toString.split("[.]")(1).size == 1)
+        x - 0.00001
+      else x
+
+    val adjustedY =
+      if (y.toString.split("[.]")(1).size == 1)
+        y + 0.00001
```

Comment:
This was for points that were exactly on the border of tile (e.g the lat or lon is exactly "10"). They needed to be moved very slightly into a tile, otherwise geotrellis would map them to 1 row/col outside the tile (e.g. 401 on a 400x400 tile). 

The if condition is just to check if it's a whole integer. Idk why I did it that hacky way, I'll change it to like Math.rint(x) == x or something. Although, I realize now this only applies to the 1x1 grid, which we don't use for fires anyway. And I'm not sure if this would be an issue for other grids (hasn't been on existing archive), since the grid size ends up being a rounded infinite float.

Reply:
For rasters, pixels which are on the top and left boarder are concidered to be inside the tile, and if they fall onto the bottom or right corner, outside.